cpp_ext/TorchTensor.cpp

```diff
@@ -389,6 +389,18 @@ void PyAnyTorchTensorValue::bindDerived(ClassTy &c) {
                          &DefaultingPyInsertionPoint::resolve());
         });
 
+  // @overload double(self) -> Tensor
+  c.def("double",
+        [](const PyAnyTorchTensorValue &self) -> PyAnyTorchTensorValue {
```

Comment:
what i was proposing is something like this

```suggestion
        [](const PyAnyTorchTensorValue &self, const PyTorch_BoolValue& non_blocking, const PyTorch_BoolValue& copy, const PyAnyTorchOptionalIntValue& memory_format, const DefaultingPyLocation& loc,  const DefaultingPyInsertionPoint& ip) -> PyAnyTorchTensorValue {
    return to(self, dtype, non_blocking, copy, memory_format, loc.get(), ip.get());
    }, "non_blocking"_a = false, "copy"_a = false, "memory_format"_a = py::none(), py::kw_only(), "loc"_a = py::none(), "ip"_a = py::none());                
```

Note the only bindings that should have `getValueLocation(self)` and `DefaultingPyInsertionPoint::resolve()` are those where you can't every pass those params (i.e. magic methods like `__add__`).